state/txns.go

```diff
@@ -53,10 +55,32 @@ type multiModelRunner struct {
 	modelUUID string
 }
 
+func shortStack() string {
+	var rv []string
+	for _, line := range strings.Split(string(debug.Stack()), "\n") {
+		if len(line) > 0 && line[0] == '\t' {
+			rv = append(rv, strings.Split(line[1:], " ")[0])
+		}
+	}
+	// We definitely have at least 3 objects in rv - debug.Stack(), this function and the one that called it.
```

Comment:
// Trim off debug.Stack() and shortStack() and the caller of shortStack from the stack trace.
I wonder if we actually want to trim the caller. RunTransaction() seems relevant to someone debugging it.